src/main/java/uk/gov/hmcts/divorce/divorcecase/model/ConditionalOrder.java

```diff
@@ -23,4 +36,77 @@
     @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS")
     private LocalDateTime dateConditionalOrderSubmitted;
 
+    @CCD(
+        label = "Link to respondent answers",
+        access = {DefaultAccess.class}
+    )
+    private Document respondentAnswersLink;
+
+    @CCD(
+        label = "Does the petitioner want to continue with the divorce and apply for a decree nisi?",
+        access = {DefaultAccess.class}
+    )
+    private YesOrNo applyForDecreeNisi;
+
+    @CCD(
+        label = "Link to online petition",
+        access = {DefaultAccess.class}
+    )
+    private Document onlinePetitionLink;
+
+    @CCD(
+        label = "Do you need to change your application or add anything?",
+        hint = "If you change or add anything which means your application has to be sent to your "
+                + "husband/wife again you may have to pay a £95 fee",
+        access = {DefaultAccess.class}
+    )
+    private YesOrNo changeOrAddToApplication;
+
+    @CCD(
+        label = "Is everything stated in this divorce petition true?",
+        access = {DefaultAccess.class}
+    )
+    private YesOrNo isEverythingInPetitionTrue;
+
+    @CCD(
+        label = "Does the applicant find it intolerable to live with the respondent?",
+        access = {DefaultAccess.class}
+    )
+    private YesOrNo doesApplicantFindItIntolerable;
+
+    @CCD(
+        label = "Date the applicant found out about the adultery",
+        typeOverride = Date,
+        access = {DefaultAccess.class}
+    )
+    @JsonFormat(pattern = "yyyy-MM-dd")
+    private LocalDate dateOfAdulteryKnowledge;
```

Comment:
This shouldn't be included any more as we've removed the reason for getting a divorce